[Editing Integration]  Add `editMode` to `/editing/config` endpoint response with configurable integration option (#1803)
Sitecore/jss: docs/upgrades/unreleased.md

    ```
    import { EditingScripts } from '@sitecore-jss/sitecore-jss-nextjs';
    ...
    const Scripts = (): JSX.Element | null => (
        <>
        <EditingScripts />
        ...
        </>
    );
    ```

> * We have introduced a new configuration option, `pagesEditMode`, in the `\src\pages\api\editing\config.ts` file to support the new metadata architecture for pages. This option allows you to specify the editing mode used by Pages. It is set to `metadata` by default. However, if you don't prefer use `metadata` and continue using the existing architecture, you can explicitly set the `pagesEditMode` to `chromes`.

Reviewer: ```suggestion
* We have introduced a new configuration option, `pagesEditMode`, in the `\src\pages\api\editing\config.ts` file to support the new editing metadata architecture for Pages (XMCloud). This option allows you to specify the editing mode used by Pages. It is set to `metadata` by default. However, if you are not ready to use a new integration and continue using the existing architecture, you can explicitly set the `pagesEditMode` to `chromes`.
```

    ```
        const handler = new EditingConfigMiddleware({
        ...

>         pagesEditMode: EditMode.Chromes,

Reviewer: Can you also, please, add an import statement for _EditMode_?

    ```
    import { EditingScripts } from '@sitecore-jss/sitecore-jss-nextjs';
    ...
    const Scripts = (): JSX.Element | null => (
        <>
        <EditingScripts />
        ...
        </>
    );
    ```

* We have introduced a new configuration option, `pagesEditMode`, in the `\src\pages\api\editing\config.ts` file to support the new editing metadata architecture for Pages (XMCloud). This option allows you to specify the editing mode used by Pages. It is set to `metadata` by default. However, if you are not ready to use a new integration and continue using the existing architecture, you can explicitly set the `pagesEditMode` to `chromes`.

    ```

>         import { EditMode } from '@sitecore-jss/sitecore-jss/layout';

Author: fixed directly on dev
